src/lib/pages/backoffice/Providers/ProviderSearch/ProviderSearch.js

```diff
@@ -0,0 +1,149 @@
+import { responseRejectInterceptor } from '@api/base';
+import { providerApi } from '@api/providers';
+import { NewButton } from '@components/backoffice/buttons/NewButton';
+import { ExportReactSearchKitResults } from '@components/backoffice/ExportSearchResults';
+import { QueryBuildHelper } from '@components/SearchBar/QueryBuildHelper';
+import {
+  invenioConfig,
+  setReactSearchKitInitialQueryState,
+  setReactSearchKitDefaultSortingOnEmptyQueryString,
+  setReactSearchKitUrlHandler,
+} from '@config';
+import history from '@history';
+import SearchAggregationsCards from '@modules/SearchControls/SearchAggregationsCards';
+import { SearchControls } from '@modules/SearchControls/SearchControls';
+import { SearchControlsOverridesMap } from '@modules/SearchControls/SearchControlsOverrides';
+import SearchFooter from '@modules/SearchControls/SearchFooter';
+import { ProviderRoutes } from '@routes/urls';
+import React, { Component } from 'react';
+import { OverridableContext } from 'react-overridable';
+import { Link } from 'react-router-dom';
+import {
+  EmptyResults,
+  Error,
+  InvenioSearchApi,
+  ReactSearchKit,
+  ResultsList,
+  ResultsLoader,
+  SearchBar,
+} from 'react-searchkit';
+import { Button, Container, Grid, Header } from 'semantic-ui-react';
+import ProviderListEntry from './ProviderListEntry';
+
+export class ProviderSearch extends Component {
+  modelName = 'PROVIDERS';
+
+  searchApi = new InvenioSearchApi({
+    axios: {
+      url: providerApi.searchBaseURL,
+      withCredentials: true,
+    },
+    interceptors: {
+      response: { reject: responseRejectInterceptor },
+    },
+  });
+
+  viewDetails = ({ row }) => {
+    return (
+      <Button
+        as={Link}
+        to={ProviderRoutes.providerDetailsFor(row.metadata.pid)}
+        compact
+        icon="info"
+        data-test={row.metadata.pid}
+      />
+    );
+  };
+
+  render() {
+    const helperFields = [
```

Comment:
Can you please fix this? it should be helping and autofilling the searchbar with real fields of the provider:
```
{
        name: 'name',
        field: 'provider.name',
        defaultValue: 'Safari'
      },
```

Reply:
This is fixed